src/screens/KeyManagement/OCR2KeysCreate.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField'
+import MenuItem from '@material-ui/core/MenuItem'
+import { ConfirmationDialog } from 'components/Dialogs/ConfirmationDialog'
+import { useOCR2KeyFamilyQuery } from 'hooks/queries/useOCR2KeysQuery'
+
+export interface Props {
+  showCreateKeyDialog: boolean
+  setToggleCreateKeyDialog: (toggleCreateKeyDialog: boolean) => void
+  onCreate: (chainType: string) => Promise<any>
+}
+
+export const OCR2KeysCreate: React.FC<Props> = ({
+  showCreateKeyDialog,
+  setToggleCreateKeyDialog,
+  onCreate,
+}) => {
+  const [createOCR2KeyType, setCreateOCR2KeyType] = React.useState<
+    string | undefined
+  >(undefined)
+
+  const handleChainTypeChange = (
+    event: React.ChangeEvent<{ value: unknown }>,
+  ) => {
+    setCreateOCR2KeyType(event.target.value as string)
+  }
+
+  const createOCR2KeyHandler = () => {
+    setToggleCreateKeyDialog(false)
+    if (createOCR2KeyType == undefined) {
+      return
+    }
+    onCreate(createOCR2KeyType)
+  }
+
+  const { loading, error, data } = useOCR2KeyFamilyQuery()
+  if (loading || error) return <div />
```

Comment:
Should this be in the beginning of the function? If we can't get data, we dont care about the rest of it's implementation anyway

Reply:
Fixed in [`e23b840` (#38)](https://github.com/smartcontractkit/operator-ui/pull/38/commits/e23b840521573d7dfa5205caadb0b54585ffd344) :)